docs/csharp/language-reference/configure-language-version.md

```diff
@@ -20,6 +20,17 @@ You can override the default behavior by specifying a language version. There ar
 - Set the language version [for multiple projects in a subdirectory](#configure-multiple-projects).
 - Configure the [`-langversion` compiler option](#set-the-langversion-compiler-option).
 
+## Required SDK and Runtime version
+
+Language features may rely on library types, or APIs that are introduced in specific versions of .NET Standard, or specific implementations of the .NET library. The following list provides information on the supported SDK version corresponding to recent language versions:
```

Comment:
I would understand "SDK version" as "version of the .Net Core SDK". But the list is not showing that, it's listing target frameworks and/or runtimes. Especially consider that e.g. .Net Core SDK 2.1.202 includes .Net Core Runtime 2.0.9, and so can't be used to target `netcoreapp2.1`.

Reply:
Good point.  I reworked the paragraph.